api/v1alpha1/storageconsumer_types.go

```diff
@@ -46,8 +45,12 @@ type StorageConsumerSpec struct {
 type CephObjectsSpec struct {
 	// BlockPoolName holds the name of created ceph block pool.
 	BlockPoolName string `json:"blockPoolName,omitempty"`
-	// CephUser holds the name of created ceph user.
-	CephUser string `json:"cephUser,omitempty"`
+	// CephRbdUser holds the name of created ceph user required for rbd.
+	CephRbdUser string `json:"cephRbdUser,omitempty"`
+	// SubVolumeGroupName holds the name of created subVolume group.
+	SubVolumeGroupName string `json:"subVolumeGroupName,omitempty"`
```

Comment:
Aren't these fields mandatory? Why do we have `omitempty`?

Reply:
These fields are updated only when they are created and not to update before creation